index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang='en'>
+    <head>
+        <meta charset="UTF-8">
+        <title>WEATHER REPORT</title>
+        <link href="./styles/index.css" rel='stylesheet'>
+        <script src="https://kit.fontawesome.com/73e38c15ac.js" crossorigin="anonymous"></script>
+
+    </head>
+    <body>
+        <div id="landscape" class="sunny">
```

Comment:
`main` would be a more appropriate element here:

```suggestion
        <main id="landscape" class="sunny">
```